oteapi/strategies/parse/excel_xlsx.py

```diff
@@ -21,7 +21,7 @@
 class XLSXParseDataModel(BaseModel):
     """Data model for retrieving a rectangular section of an Excel sheet."""
 
-    worksheet: str = Field(..., description="Name of worksheet to load.")
+    worksheet: str = Field("Sheet1", description="Name of worksheet to load.")
```

Comment:
To set a default value? Will this value not change depending on the origin language for the created excel sheet? Like `Ark1` in Danish? Maybe this should be added to the description? Or perhaps instead some form of post-processing could be done to auto-choose the first sheet found in the excel file?